### ISSUE TITLE
add mount.fuse

### ISSUE THREAD

**Issue by ederuiter** [coreos/coreos-overlay#1518]
Currently coreos ships with the fuse file-system enabled, but it doesn't have the mount.fuse helper. This makes it hard to mount fuse based file-systems.
An alternative would be to build mount with --enable-fs-paths-extra=/opt/bin so that mount helpers could be installed in /opt/bin (currently only /sbin/, /sbin/fs.d/, /sbin/fs, /usr/sbin/, /bin/ and /usr/bin are searched for helpers and all those locations are mounted readonly)

**Comment by mischief** [comment]
see coreos/bugs#358. have you tried running mount.fuse in a container?

**Comment by ederuiter** [comment]
Yes I did,  but that is not an option as fuse based file-systems cannot be exported from a container to the host. And I need to access this filesystem in several containers (and running it in all containers is not an option as our filesystem replicates all data to local storage)
